django_multitenant/tests/settings.py

```diff
@@ -35,10 +35,12 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "django_multitenant.middlewares.MultitenantMiddleware",
```

Comment:
Django itself has middleware, should we be more clear in the name of this one that it's specific to DRF?

Reply:
I think we can add more functions into this Middleware. Right now, we are using it for DRF, but we may have additional features as well. 
Additionally, for this middleware, there is no DRF dependency. It processes all requests. Therefore, some may use this middleware in another framework, such as Flask, since this Middleware processes standard request; not a specialized DRF request

Reply:
django_multitenant.middlewares.MultitenantMiddleware
isn't calling unset_current_tenant()
Am I missing anything?

The unsetting of the tenant is essential because of how webservers work
     Since the tenant is set as a thread local, the thread is not killed after the request is processed
     So after processing of the request, we need to ensure that the tenant is unset
     Especially required if you have public users accessing the site

     This is also essential if you have admin users not related to a tenant.